app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -14,21 +14,26 @@ import DocumentSideAcuantCapture from './document-side-acuant-capture';
 import DocumentCaptureNotReady from './document-capture-not-ready';
 import { FeatureFlagContext } from '../context';
 import DocumentCaptureAbandon from './document-capture-abandon';
+import { DocumentCaptureSubheaderOne, SelfieStepWithHeader } from './documents-step';
 
 interface DocumentCaptureReviewIssuesProps {
   isFailedDocType: boolean;
   remainingAttempts: number;
   captureHints: boolean;
   registerField: RegisterFieldCallback;
-  value: { string: Blob | string | null | undefined } | {};
+  value: {
+    front?: Blob | string | null | undefined;
+    back?: Blob | string | null | undefined;
+    selfie?: Blob | string | null | undefined;
+  };
```

Comment:
Can you say more about this change? I think I'm just not following what it gives us, or why we removed `selfie` from DocumentSides below. (I think this is probably a good change, just asking for my own curiosity / learning since I'm not following.)

Reply:
A good question. I had to modify more than I expected in the typescript. What was in there:
`  value: { string: Blob | string | null | undefined } | {};`
- I'm pretty sure that this says "When there is a property named "string" it can have these types" I think that's probably not what the author meant to do.
- It's probably meant to say "Any property on this object can have these types".
- Instead of doing that, I made it more specific to the properties we actually expect.
- Now it says something like "front, back, and selfie are all optional, but can all have any of these types".

Reply:
I'm going to follow up on this in a later PR rather than expanding the scope of this one. Brittany and I also chatted about this.